scipy/stats/_multivariate.py

```diff
@@ -847,16 +904,20 @@ def rvs(self, alpha, size=1):
 
         """
         alpha = _dirichlet_check_parameters(alpha)
-        return np.random.dirichlet(alpha, size=size)
+        if random_state is None:
+            return self.random_state.dirichlet(alpha, size=size)
```

Comment:
@ev-br Not sure I understand, self.random_state calls the property `random_state` which returns the private `self._random_state`. Do you want me to directly access `self._random_state`?